### ISSUE TITLE
ansible-playbook ignores --step

### ISSUE THREAD

**Issue by sheldonh** [ansible/ansible#11667]
##### Issue Type:

Bug Report
##### Ansible Version:

```
ansible 2.0.0 (devel 7124504ccf) last updated 2015/07/21 11:36:31 (GMT +200)
  lib/ansible/modules/core: (detached HEAD 291fef3b34) last updated 2015/07/17 15:26:39 (GMT +200)
  lib/ansible/modules/extras: (detached HEAD ff2386faf4) last updated 2015/07/17 15:26:43 (GMT +200)
  v1/ansible/modules/core: (detached HEAD f8d8af17cd) last updated 2015/07/17 15:26:48 (GMT +200)
  v1/ansible/modules/extras: (detached HEAD 495ad450e5) last updated 2015/07/17 15:26:52 (GMT +200)
  configured module search path = None
```
##### Ansible Configuration:

None
##### Environment:

N/A
##### Summary:

The `--step` flag to `ansible-playbook` is ignored.
##### Steps To Reproduce:

Create this `playbook.yml`:

``` yaml

---
-
  hosts: localhost
  connection: local
  gather_facts: false
  tasks:
    - debug: msg="task 1"
    - debug: msg="task 2"
```

Run it as follows:

``` shell
ansible-playbook -i /dev/null --step playbook.yml
```
##### Expected Results:

As in 1.9.2, `ansible-playbook` should prompt before executing each task, e.g.

```
PLAY [localhost] ************************************************************** 
Perform task: debug msg="task 1" (y/n/c): 
```
##### Actual Results:

The playbook is run as if `--step` was not specified:

```
PLAY ****************************************************************************

TASK [debug msg=task 1] *********************************************************
ok: [localhost] => {
    "changed": false, 
    "msg": "task 1"
}

TASK [debug msg=task 2] *********************************************************
ok: [localhost] => {
    "changed": false, 
    "msg": "task 2"
}

PLAY RECAP **********************************************************************
localhost                  : ok=2    changed=0    unreachable=0    failed=0   
```

**Comment by bcoca** [comment]
this has not yet been implemented

**Comment by sheldonh** [comment]
No worries. Just remember to take it out of `usage` if it doesn't make it in for the release. ;-)

**Comment by bcoca** [comment]
it is required for the release

**Comment by bcoca** [comment]
fixed via 88e1ef8c9bc6434786ce10dbf0ed46a5b2448c1c
